File: common/src/main/java/org/valkyrienskies/clockwork/platform/Dist.java
package org.valkyrienskies.clockwork.platform;

import com.tterrag.registrate.util.entry.EntityEntry;
import dev.architectury.injectables.annotations.ExpectPlatform;
import org.valkyrienskies.clockwork.content.curiosities.tools.bluperglue.BluperGlueEntity;

public class Dist {

    @ExpectPlatform
    public static void onClient(Runnable runnable) {
        throw new AssertionError();
    }

    @ExpectPlatform
    public static void onDedicatedServer(Runnable runnable) {
        throw new AssertionError();
    }
}

File: common/src/main/kotlin/org/valkyrienskies/clockwork/platform/Dist.kt
package org.valkyrienskies.clockwork.platform

import dev.architectury.injectables.annotations.ExpectPlatform

object Dist {
    @ExpectPlatform
    @JvmStatic
    fun onClient(runnable: Runnable) {
        throw AssertionError()
    }

    @ExpectPlatform
    @JvmStatic
    fun onDedicatedServer(runnable: Runnable) {
        throw AssertionError()
    }
}